exporters/otlp/internal/otlpconfig/options.go

```diff
@@ -49,22 +49,7 @@ const (
 		"name":[
 			{ "service":"opentelemetry.proto.collector.metrics.v1.MetricsService" },
 			{ "service":"opentelemetry.proto.collector.trace.v1.TraceService" }
-		],
-		"retryPolicy":{
-			"MaxAttempts":5,
-			"InitialBackoff":"0.3s",
-			"MaxBackoff":"5s",
-			"BackoffMultiplier":2,
-			"RetryableStatusCodes":[
-				"CANCELLED",
-				"DEADLINE_EXCEEDED",
-				"RESOURCE_EXHAUSTED",
-				"ABORTED",
-				"OUT_OF_RANGE",
-				"UNAVAILABLE",
-				"DATA_LOSS"
-			]
-		}
+		]
```

Comment:
Is the `DefaultServiceConfig` needed anymore?

Reply:
Nope, just removed it, nice catch :)